config/cloud-service.js

```diff
@@ -0,0 +1,34 @@
+// Import the required module
+const cloudService = require('client-cloud-services');
+
+/**
+ * Function to initialize the cloud client with the provided configuration.
+ * If any error occurs during initialization, it will be logged, and the process will exit.
+ */
+function initializeCloudClient() {
+  try {
+    // Cloud configuration containing credentials and container names
+    let cloudConfig = {
+      provider: process.env.CLOUD_STORAGE_PROVIDER,
```

Comment:
@borkarsaish65 the new envs we should also add to envVariables.js file and env samples file. Please refer elevate-project service if you need better clarity

Reply:
Done